tests/ops/op_math/test_composite.py

```diff
@@ -101,14 +101,19 @@ def test_parameters(self):
     def test_data(self):
         """Test that data is initialized correctly."""
         op = ValidOp(qml.RX(9.87, wires=0), qml.Rot(1.23, 4.0, 5.67, wires=1), qml.PauliX(0))
-        assert op.data == [9.87, 1.23, 4.0, 5.67]
+        assert op.data == (9.87, 1.23, 4.0, 5.67)
 
     def test_data_setter(self):
         """Test the setter method for data"""
         op = ValidOp(qml.RX(9.87, wires=0), qml.Rot(1.23, 4.0, 5.67, wires=1), qml.PauliX(0))
-        assert op.data == [9.87, 1.23, 4.0, 5.67]
+        assert op.data == (9.87, 1.23, 4.0, 5.67)
 
-        new_data = [1.23, 0.0, -1.0, -2.0]
+        new_data = (
+            1.23,
+            0.0,
+            -1.0,
+            -2.0,
```

Comment:
This tuple should just be on a single line. I assume that you left the comma at the end and when you ran `black` it reformatted the tuple.